examples/ssr/src/app.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import { Routes, Route, Outlet, Link } from "react-router-dom";
+
+export default function App() {
+  return (
+    <div>
+      <h1>Welcome to the app!</h1>
+
```

Comment:
Maybe add a `<p>` here telling them why this is cool. Go look at the HTML source!